types/ramda/test/forEach-tests.ts

```diff
@@ -0,0 +1,26 @@
+import * as R from 'ramda';
+
+(() => {
+  function printXPlusFive(x: number) {
+    console.log(x + 5);
+  }
+
+  R.forEach(printXPlusFive, [1, 2, 3]);
+})();
+
+// (() => {
+//   function printXPlusFive(x, i) { console.log(i + 5); }
+//   R.forEach.idx(printXPlusFive, [{name: 1}, {name: 2}, {name: 3}]);
+// })();
```

Comment:
is there reason to keep this around?

Reply:
I think so. This actually tests being able to pass in an array of objects rather than just an array of numbers, so it's slightly different.

I didn't want to uncomment it right now, but I could see how it could be useful to remind us of other test-cases once we get around to writing more later on.

I'm less concerned about this commented-out test, and more concerned with the tests below it (lines 16-26), which appear to be a copy of the first uncommented test-case (lines 3-9) except with the addition of the curried test-case. I had removed other duplicate tests for other functions, so I'm not 100% sure why I left that one behind.

Still, no harm no foul. We can clean them up further later-on.